index.ts

```diff
@@ -16,8 +16,11 @@ const program = new Command(packageJson.name)
   .option('-d, --dry', 'Dry run (no changes are made to files)')
   .option('-p, --print', 'Print transformed files to stdout')
   .option('--verbose', 'Show more information about the transform process')
+  .option('--silent', "Don't print anything to stdout")
```

Comment:
Necessary for the tests to avoid extra noise, and it's a good option provided by jscodeshift.